### ISSUE TITLE
Would PRs to this repository count towards hacktoberfest 2021?

### ISSUE THREAD

**Issue by basic-bhavya** [Prachi8282/Java-Algorithmic-Challenges#2]

**Comment by Prachi8282** [comment]
@basic-bhavya Yes genuine PR's will count towards Hacktoberfest 2021. Please read the Readme, Code of Conduct and the Contributing guide and you can start contributing! 😀✌️

**Comment by Prachi8282** [comment]
@basic-bhavya Just create a PR containing the solution to an Algorithmic Problem and I will merge it in due time.
